packages/core/src/Utils/Language/language.js

```diff
@@ -25,5 +25,8 @@ export const getURL = lang => urlForLanguage(lang);
 export const changeLanguage = key => {
     SocketCache.clear();
 
+    if (key === 'EN') {
```

Comment:
why are we setting this only for `EN`?

and in `getUrlSmartTrader` function we use small case for `EN`. which is correct?
```
const i18n_language = window.localStorage.getItem('i18n_language') || 'en';```

Reply:
We want to manually set this param to `English` because the param will be missing from URL for english language. It will be shown and set for other languages from URL, so we will not need to set it twice manually. Regarding the case, the key is always capital case, so it's supposed to be capital case as well. I think the one in getUrlSmartTrader is incorrect.

Reply:
SmartTrader sets language as a URL parameter, rather than a query param, that's why there is a difference in implementation. I believe we want to keep lower case for url params.